guide/english/css/css-performance/index.md

```diff
@@ -32,6 +32,7 @@ As a rule of thumb:
 * Don't shy away from using a lot of descriptive class names `.footer-nav-link`
 * Try to use the child selector instead if you want to select a direct child of an element `.image-container > img`
 * Avoid using universal selectors `* { }`
+* Remove unnecessary white spaces and consider minifying your CSS
```

Comment:
Only a suggestion but consider adding more detail towards this, as to how someone may minify their CSS, maybe add tool links which automate this process.

Reply:
@co16353sidak I think this can be merged as is.  Feel free to create a new PR to add the suggestions you have made.